api/hasura/actions/_handlers/createOrgMembers.ts

```diff
@@ -86,6 +86,41 @@ async function handler(req: VercelRequest, res: VercelResponse) {
     );
   }
 
+  //check if names are used by other coordinape users
+  const { profiles: existingNames } = await adminClient.query(
+    {
+      profiles: [
+        {
+          where: {
+            _or: users.map(user => {
+              return {
+                _and: [
+                  { name: { _eq: user.name } },
+                  { address: { _nilike: user.address } },
```

Comment:
if the address DOES NOT match, but the name does, we still have an error though right? @sshmm

Reply:
should this just be to check if any profile name is already in use, regardless of address?

Reply:
nevermind - this is correct.

Reply:
May be that user already has the a profile for this address so if the name and address matches the profile it won't throw an error.
What it does is that it fetches  all profiles with names from the input as long as these names are assigned for profiles with different addresses from the input addresses.